deployments/chef/recipes/default.rb

```diff
@@ -24,13 +24,34 @@
 
 if platform_family?('debian')
   include_recipe 'signalfx_agent::deb_repo'
-elsif platform_family?('rhel', 'amazon', 'fedora')
+elsif platform_family?('rhel', 'amazon', 'fedora', 'suse', 'opensuse')
   include_recipe 'signalfx_agent::yum_repo'
 end
 
 case node['platform_family']
 when 'windows'
   include_recipe 'signalfx_agent::win'
+when 'suse', 'opensuse'
+  zypper_package 'install-deps' do
+    package_name %w(libcap2 libcap-progs libpcap1 shadow)
+  end
+  tmpdir = Dir.mktmpdir
+  package_name = if node['signalfx_agent']['package_version'].nil?
+                   'signalfx-agent'
+                 else
+                   "signalfx-agent-#{node['signalfx_agent']['package_version']}"
+                 end
+  execute 'download-agent' do
+    command "zypper --pkg-cache-dir=#{tmpdir} download #{package_name}"
+  end
+  execute 'install-agent' do
```

Comment:
This needs conditional logic so it doesn't do this every time chef runs.  Same thing with the download command above.

Reply:
Added logic to detect installed and latest versions, and only download/install when there is a mismatch.